src/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+
+import Page from "./components/Page";
+
+function App() {
+  const [state, setState] = useState({
+    count: 0
+  });
+
+  const { count } = state;
+
+  // click me를 눌럿을때는 1만들어오게 하고
+  // 나머지는 숫자만큼 들어오게 해서 더해준다.
+  function handleClick(value) {
+    setState({
+      count: count + value
+    });
+  }
+
+  return <Page count={count} onClick={handleClick} />;
```

Comment:
```suggestion
  return (
    <Page 
      count={count}
      onClick={handleClick} 
     />
   );
```

저는 `return` 을 작성할 때 이렇게 기본 형태를 잡아주는 편이에요.
컴포넌트도 props 가 2개이상이면 줄바꿈을 해서 코드 가독성을 높여주는 편이에요.

어떤게 더 나은 것 같나요?

Reply:
그렇게 하려고 했는데 eslint가 잘못 설정되어있는지 저장을 누르면 한줄로 줄어드는데 이건 어떻게 해결할 수 있을까요?

Reply:
저도 그렇게 하고 싶었는데 eslint 설정이 잘못되었는지 저장을 누르면 한줄로 바꿔지는데 

이건 어떻게 해결할 수 있을까요>?

Reply:
저장 눌렀을 때 바뀌는건 에디터 설정때문일거에요. 에디터 설정을 한 번 찾아보시면 좋을 것 같아요.